website/content/gateway/api_reference/resources/debit_refunds/debit_refunds.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "debit_refunds"
+date: 2023-01-05T13:37:00+01:00
+anchor: "debit_refunds"
+weight: 178
+---
+#### Debit refunds
+To refund money to a cardholder's bank account that has been transferred by a debit you make a new refund resource. Only a single refund can be made for a debit and it can only be for the full amount.
+
+```shell
+POST https://gateway.clearhaus.com/debits/:id/refunds
+```
+
+##### Parameters
+No parameters are needed to make a new refund transaction.
+
+{{% notice %}}
+**Notice**: A refund of a Visa AFT debit might be declined by Visa or the issuer if the debit age is 24 hours or more, or if settlement has taken place for the debit. The transaction gateway does not track debit age or settlement, i.e. it relies on Visa and the issuer to decline late refunds.
```

Comment:
Continuing from https://github.com/clearhaus/gateway-api-docs/pull/146#pullrequestreview-1238058763, I suspect "clearing" is what's meant?

```suggestion
**Notice**: A refund of a Visa AFT debit might be declined by Visa or the issuer if the debit age is 24 hours or more, or the debit has cleared. The transaction gateway does not track the clearing of the debit, i.e. it relies on Visa and the issuer to decline late refunds.
```

Do we decline debit refunds when the debit is "too old" — say, the normal ~180 days?

Reply:
That is a good question. It wasn't my plan as I hope to simplify things by letting upstream be the judge :balance_scale: :hammer: 

We might want to implement a check if it's expensive in fees for us to make very late refunds.

@clearhaus/transaction-compliance Do you have any input here?

Reply:
On another note, we want to limit the amount of time we need to store the PAN. The PAN isn't sent to us in a debit refunds request.

Reply:
I also tend to say the right word here is "clearing", not "settlement". So I'm inclined to apply your suggestion. I will let the thread hang a bit before doing so.

Reply:
I've applied your suggestion regarding settlement vs clearing. It seems more correct than the original text, and we can change our minds later.

I will move on as we can introduce decline of late debit refunds later.